app/models/dataload_job.rb

```diff
@@ -0,0 +1,26 @@
+class DataloadJob < ApplicationRecord
+  belongs_to :vendor
+  enum :status, [
+    :scheduled, # the initial value when a job is created
+    :queued, # the status set when the API has reported the job
+    :in_progress, # when airflow has started the job it will update this value via the API
+    :failed, # if the airflow job fails for any reason, it should update this value via the API
+    :completed # when the job completes successfully in airflow, this value will be updated via the API
+  ]
+  enum :recurrence, [
+    :no_recurrence,
+    :daily_recurrence,
+    :weekly_recurrence,
+    :monthly_recurrence,
+    :yearly_recurrence
+  ]
+
+  # date: now (if immediate), the date selected for one-off (recurrence none), the next recurrence date (*see below)
+  # dataload_profile_id: The id of the folio data load profile selected
+  # created_by: The sunetid of the user creating the job
+  # additional_contacts: Array of the sunetid(s) of any additional users to be notified of job state in the data pipeline.
+  # filename (optional): The filename of a manually uploaded file
+  # file_pattern (optional): The filename pattern for airflow to use in order to find the correct vendor file
+  # processing_tasks: Array of selected known data processing tasks.
+  # log (large text field to capture any provided log data from airflow on failure)
```

Comment:
is this comment pertaining to a to-be-implemented ticket?  If so, maybe reference the ticket?  Or explain what this comment section is about?

Reply:
These are notes on the model fields. I added an additional comment to make clearer.